hack/render-manifests.go

```diff
@@ -14,25 +15,40 @@ func exitWithError(err error, cause string, args ...interface{}) {
 	os.Exit(1)
 }
 
+const NumArgs = 5
+
+const (
+	HandlerNamespaceIdx = iota
+	HandlerImageIdx
+	HandlerPullPolicyIdx
+	InputDirIdx
+	OutputDirIdx
+)
+
 func main() {
 	type Inventory struct {
-		Namespace           string
+		HandlerNamespace    string
 		NMStateHandlerImage string
-		ImagePullPolicy     string
+		HandlerPullPolicy   string
+		HandlerPrefix       string
 	}
 
-	if len(os.Args) < 6 {
-		fmt.Fprintf(os.Stderr, "usage: go run render-manifests.go [namespace] [image] [pull-policy] [input-dir] [output-dir]\n")
+	handlerPrefix := flag.String("handler-prefix", "", "Name prefix for the handler's resources")
+	flag.Parse()
+
+	if flag.NArg() < NumArgs {
+		fmt.Fprintf(os.Stderr, "usage: go run render-manifests.go [-handler-prefix handler-prefix] <handler-namespace> <handler-image> <handler-pull-policy> <input-dir> <output-dir>\n")
 		os.Exit(1)
 	}
 
 	inventory := Inventory{
-		Namespace:           os.Args[1],
-		NMStateHandlerImage: os.Args[2],
-		ImagePullPolicy:     os.Args[3],
+		HandlerNamespace:    flag.Arg(HandlerNamespaceIdx),
+		NMStateHandlerImage: flag.Arg(HandlerImageIdx),
```

Comment:
Rename to `HandlerImage` for consistency, It also means renaming the template token.